### DISCUSSION TITLE
Allow dragging multiple elements into an @exported array field to add them at once

### DISCUSSION THREAD

**Discussion by Kartopod** [godotengine/godot-proposals#8096] [Editor]
Dragging multiple elements onto the add element button (Or maybe the array field itself, or both) should add all the elements to the array at once. This would make adding elements easier. At the moment you would have to manually drag every single element one by one. 
![image](https://github.com/godotengine/godot-proposals/assets/75985716/7360f61f-ae1a-40cf-9b02-50de67ff83b1)

**Comment by Mickeon** [comment]
I've instinctually tried this in the past and was left a bit disappointed, so it would definitely be nice to have.

**Comment by jmarceno** [comment]
I think I've seem a proposal for something similar recently, but I can´t find it, although I found #6862 that asks for the ability to drag and drop single elements.
